gender_novels/analysis/analysis.py

```diff
@@ -14,13 +17,72 @@ def test_function():
 
     :return:
     """
-
+    '''
```

Comment:
Is this meant to be part of the doctest? If it's a comment, can you comment it out instead of setting it as an anonymous string

Reply:
i'm not too sure, it had been there originally (as a placeholder?) and I left it in case someone else needed it. But it's probably just a placeholder, I'll just delete it.